server/services/store/storetests/initialize.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package storetests
+
+import (
+	"testing"
+	"github.com/mattermost/focalboard/server/services/store"
+	"github.com/stretchr/testify/require"
+)
+
+
+func StoreTestInitializeStore(t *testing.T, setup func(t *testing.T) (store.Store, func())) {
+	t.Run("Initialize Template", func(t *testing.T) {
+		store, tearDown := setup(t)
+		defer tearDown()
+		testInitializeTemplate(t, store)
+
+	})
+
+}
+
+func testInitializeTemplate(t *testing.T, store store.Store) {
+        t.Run("InitializeTemplate",func(t *testing.T) {
+
+	err := store.InitializeTemplates()
+	require.NoError(t, err)
```

Comment:
We should also validate that the templates were indeed inserted into the database

Reply:
Hey thanks, I'll try to figure out how to do that out and re-submit

Reply:
Hey, I'm a bit confused with the validation part could you let me know if this would be the right way?

```
package storetests

import (
        "github.com/mattermost/focalboard/server/services/store"
        "github.com/stretchr/testify/require"
        "testing"
)

func StoreTestInitializeStore(t *testing.T, setup func(t *testing.T) (store.Store, func())) {
        container := store.Container{
                WorkspaceID: "0",
        }
        t.Run("Initialize Template", func(t *testing.T) {
                store, tearDown := setup(t)
                defer tearDown()
                testInitializeTemplate(t, store, container)

        })

}

func testInitializeTemplate(t *testing.T, store store.Store, container store.Container) {
        t.Run("InitializeTemplate", func(t *testing.T) {
                err := store.InitializeTemplates()
                require.NoError(t, err)

                _, errBlocks := store.GetAllBlocks(container)
                require.NoError(t, errBlocks)
                //err := store.GetBlock(c store.Container, block
        })

}
```

Reply:
@harshilsharma63 @sbishel need some guidance with the code here I'm not sure how to replicate the output on my end.

Reply:
@afro-coder apologies for the delayed response. That looks fine - simply querying for all blocks and verifying the retrieved data 👍 

p.s., Feel free to hop on https://community-daily.mattermost.com/core/channels/focalboard if a message goes un-replied or you need something quickly :)

Reply:
Oh sure no worries, I'll keep that in mind!

Reply:
These CI tests failing don't specify an exact file